contrib/extract_image_id.sh

```diff
@@ -20,7 +20,13 @@ test -e $tar_path
 
 # Extracts the manifest.json file from the image's tarball
 # and finds the image id in it
-tar -xf $tar_path "manifest.json"
+if tar -xf $tar_path "manifest.json"
```

Comment:
can you do the if something then: in one line?

Reply:
Changed